Cairo Desktop/CairoDesktop.Common/Localization/Language.cs

```diff
@@ -1615,6 +1615,310 @@ static class Language
+            { "sSettings_RestartCairo", "重新啟動 Cairo 以套用更改" },
+            { "sSettings_Restarting", "重新啟動中..." },
+            { "sSettings_General_UpdateCheck", "自動檢查更新" },
+            { "sSettings_General_RunAtLogOn", "登入時自動啟動 Cairo" },
+            { "sSettings_General_Language", "語言: " },
+            { "sSettings_General_Theme", "主題: " },
+            { "sSettings_General_ThemeTooltip", "請將主題檔案放置在 Cairo 安裝路徑下。" },
+            { "sSettings_General_TimeFormat", "時間格式: " },
+            { "sSettings_General_TimeFormatTooltip", "您可以在這裡使用一些格式組件來自定義你的時間格式:\r\nd\t日\r\nddd\t星期\r\nh\t時 (12 小時制, 1 到 2 位數)\r\nhh\t時 (12 小時制, 固定 2 位數)\r\nH\t時 (24 小時制, 1 到 2 位數)\r\nHH\t時 (24 小時制, 固定 2 位數)\r\nm\t分 (1 到 2 位數)\r\nmm\t分 (固定 2 位數)\r\ns\t秒 (1 到 2 位數)\r\nss\t秒 (固定 2 位數)\r\ntt\t顯示上午/下午 (搭配 12 小時制的組件來使用)" },
+            { "sSettings_General_DateFormat", "日期格式: " },
+            { "sSettings_General_DateFormatTooltip", "您可以在這裡使用一些格式組件來自定義你的日期格式:\r\nd\t日期 (1 到 2 位數)\r\ndd\t日期 (固定 2 位數)\r\nddd\t星期 (簡短寫法)\r\ndddd\t星期 (完整寫法)\r\nM\t月份 (1 到 2 位數)\r\nMM\t月份 (固定 2 位數)\r\nMMM\t月份 (簡短寫法)\r\nMMMM\t月份 (完整寫法)\r\nyy\t年份 (簡短寫法)\r\nyyyy\t年份 (完整寫法)" },
+            { "sSettings_General_FilesAndFolders", "檔案和資料夾" },
+            { "sSettings_General_ShowFileExtensions", "顯示檔案副檔名" },
+            { "sSettings_General_ForceSoftwareRendering", "強制使用軟體渲染" },
+            { "sSettings_General_FileManager", "檔案總管: " },
+            { "sSettings_General_FoldersOpenDesktopOverlay", "使用桌面覆蓋介面取代檔案總管來開啟資料夾" },
+            { "sSettings_MenuBar_DefaultProgramsCategory", "預設應用程式分類: " },
+            { "sSettings_MenuBar_EnableMenuBarAutoHide", "自動隱藏選單列" },
+            { "sSettings_MenuBar_EnableMenuBarShadow", "啟用選單列的陰影效果" },
+            { "sSettings_MenuBar_MenuBarEdge", "選單列位置: " },
+            { "sSettings_MenuBar_NotificationArea", "通知區域" },
+            { "sSettings_MenuBar_EnableNotificationArea", "啟用通知區域介面" },
+            { "sSettings_MenuBar_NotificationAreaTaskbarWarning", "由於 Windows 工作列正在顯示，Cairo 通知區域介面將自動隱藏。如果要使用 Cairo 通知區域，請啟用 Cairo 工作列並重新啟動 Cairo。" },
+            { "sSettings_MenuBar_ShowNotifyIcons", "顯示通知區域圖示" },
+            { "sSettings_MenuBar_ShowNotifyIconsCollapsed", "摺疊" },
+            { "sSettings_MenuBar_ShowNotifyIconsAlways", "總是顯示" },
+            { "sSettings_MenuBar_CollapsibleIcons", "可隱藏圖示: " },
+            { "sSettings_MenuBar_PinnedIcons", "已釘選圖示: " },
+            { "sSettings_MenuBar_NotificationAreaPinHelp", "要使圖示固定顯示在通知區域內，請將其拖曳至 \"已釘選圖示\" 區；要使圖示在無通知時自動隱藏，則將其拖曳至 \"可隱藏圖示\" 區。" },
+            { "sSettings_MenuBar_EnableCairoMenuHotKey", "啟用 Cairo 選單快捷鍵: " },
+            { "sSettings_MenuBar_EnableMenuBarBlur", "選單列背景模糊" },
+            { "sSettings_MenuBar_EnableMenuBarMultiMon", "在所有螢幕上顯示選單列" },
+            { "sSettings_MenuBar_ShowHibernate", "在 Cairo 主選單中顯示 \"休眠\" 選項" },
+            { "sSettings_MenuBar_ShowStacksRemovableDrives", "在常用堆疊中顯示可卸除的裝置" },
+aero            { "sSettings_MenuBar_ProgramsMenuLayout", "\"應用程式\"選單配置: " },
```

Comment:
Not sure how `aero` got here :)

Reply:
I don't know why an "aero" on there

It didn't exist when I tested last night

Nevermind, it fixed.